posix.mak

```diff
@@ -10,6 +10,7 @@
 #
 
 include osmodel.mak
+PWD=$(realpath .)
```

Comment:
`realpath` resolves symlinks and unnecessarily exposes implementation details in log files or even built files. We should get rid of it and just use `$(pwd)` instead.